src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/UpdateBlurb/UpdateBlurb.tsx

```diff
@@ -10,34 +10,36 @@ import {
 
 const UpdateBlurb = ({
   currentPlan,
+  planUserCount,
   selectedPlan,
   newPlanName,
   seats,
   nextBillingDate,
 }: {
-  currentPlan?: z.infer<typeof PlanSchema>
+  currentPlan?: PlanName
```

Comment:
Keeping this as the Plan object would actually open us up a lot better than "downscaling" to PlanName, I have a similar update in the isFreePlan I believe that can hopefully help here